### ISSUE TITLE
Error when executing 'jabba ls-remote'

### ISSUE THREAD

**Issue by adrianbrink** [shyiko/jabba#56]
This warning gets triggered when executing ```jabba ls-remote``` .

```
➜  ~ jabba ls-remote
Get https://github.com/shyiko/jabba/raw/master/index.json: x509: certificate signed by unknown authority
```

System:
macOS10.12 using zsh

**Comment by shyiko** [comment]
Hi Adrian,

what does "curl -IL https://github.com/shyiko/jabba/raw/master/index.json" return? 
Chances are you are missing proper certs (see [resetting the keychain](https://www.macissues.com/2015/09/10/fix-persistent-invalid-certificate-errors/)).

**Comment by adrianbrink** [comment]
```
HTTP/1.1 302 Found
Server: GitHub.com
Date: Sat, 25 Feb 2017 08:46:22 GMT
Content-Type: text/html; charset=utf-8
Status: 302 Found
Cache-Control: no-cache
Vary: X-PJAX, Accept-Encoding
X-RateLimit-Limit: 100
X-RateLimit-Remaining: 100
Access-Control-Allow-Origin: https://render.githubusercontent.com
Location: https://raw.githubusercontent.com/shyiko/jabba/master/index.json
X-UA-Compatible: IE=Edge,chrome=1
X-Runtime: 0.024562
Content-Security-Policy: default-src 'none'; base-uri 'self'; block-all-mixed-content; child-src render.githubusercontent.com; connect-src 'self' uploads.github.com status.github.com collector.githubapp.com api.github.com www.google-analytics.com github-cloud.s3.amazonaws.com wss://live.github.com; font-src assets-cdn.github.com; form-action 'self' github.com gist.github.com; frame-ancestors 'none'; img-src 'self' data: assets-cdn.github.com identicons.github.com collector.githubapp.com github-cloud.s3.amazonaws.com *.githubusercontent.com; media-src 'none'; script-src assets-cdn.github.com; style-src 'unsafe-inline' assets-cdn.github.com
Strict-Transport-Security: max-age=31536000; includeSubdomains; preload
Public-Key-Pins: max-age=5184000; pin-sha256="WoiWRyIOVNa9ihaBciRSC7XHjliYS9VwUGOIud4PB18="; pin-sha256="RRM1dGqnDFsCJXBTHky16vi1obOlCgFFn/yOhI/y+ho="; pin-sha256="k2v657xBsOVe1PQRwOsHsw3bsGT2VzIqz5K+59sNQws="; pin-sha256="K87oWBWM9UZfyddvDfoxL+8lpNyoUB2ptGtn0fv6G2Q="; pin-sha256="IQBnNBEiFuhj+8x6X8XLgh01V9Ic5/V3IRQLNFFc7v4="; pin-sha256="iie1VXtL7HzAMF+/PVPR9xzT80kQxdZeJ+zduCB3uj0="; pin-sha256="LvRiGEjRqfzurezaWuj8Wie2gyHMrW5Q06LspMnox7A="; includeSubDomains
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-Served-By: f0ee042be143fcba78041fc2f69c0aa7
Age: 0
X-GitHub-Request-Id: C277:40F3:123A9F8:1CF9FBB:58B1445E

HTTP/1.1 200 OK
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
ETag: "1888240900ae321ade2dfadd9cf2e7cb058f6db5"
Content-Type: text/plain; charset=utf-8
Cache-Control: max-age=300
X-Geo-Block-List:
X-GitHub-Request-Id: FB08:74B7:9EC2C3:A4E135:58B1445F
Content-Length: 75630
Accept-Ranges: bytes
Date: Sat, 25 Feb 2017 08:46:24 GMT
Via: 1.1 varnish
Connection: keep-alive
X-Served-By: cache-bma7034-BMA
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1488012383.914945,VS0,VE150
Vary: Authorization,Accept-Encoding
Access-Control-Allow-Origin: *
X-Fastly-Request-ID: 03660ff44fa3bda2e31049b0b4c13b23f84b72cc
Expires: Sat, 25 Feb 2017 08:51:24 GMT
Source-Age: 0
```

**Comment by shyiko** [comment]
Thank you. 

I think I found the issue (https://github.com/golang/go/issues/14514). It seems like it has been fixed in go1.8. Can you please try the build below?

`curl -sL https://github.com/shyiko/jabba/releases/download/unstable/jabba-0.6.0-43-gcd7d6eb-darwin-amd64 > ~/.jabba/bin/jabba && chmod a+x ~/.jabba/bin/jabba`

and then `jabba ls-remote` as before.

**Comment by shyiko** [comment]
Yeah, and if that build doesn't work either then

`curl -sL https://github.com/shyiko/jabba/releases/download/unstable/jabba-unstable-darwin-amd64 > ~/.jabba/bin/jabba && chmod a+x ~/.jabba/bin/jabba`

**Comment by shyiko** [comment]
Fixed in 0.6.1. Decided to go with `hashicorp/go-rootcerts` to avoid `cgo`.
